app/subscriber/src/features/home/Home.tsx

```diff
@@ -45,7 +46,7 @@ export const Home: React.FC = () => {
   React.useEffect(() => {
     // stops invalid requests before filter is synced with date
     // wait for userinfo incase applying previously viewed filter
-    if (!!featuredStoryActionId && !!userInfo) {
+    if (!!featuredStoryActionId && !!active) {
```

Comment:
userInfo would ensure the user has been activated.  Is there anything related to user preferences in this request?

Reply:
`active` relies on `userInfo` in `FilterOptions.tsx`, the previous fix was working locally, but on dev there was some sort of weird race condition. Active will become truthy once `userInfo` is parsed and the active filter is set

Reply:
I could keep userInfo in there in the check though, just had to add active to get rid of that weird behaviour I was observing earlier

Reply:
After reading this again, I think what would be better is adding `hasProcessedInitialPreference` to a context provider, rather than the string state. Will update

Reply:
Just make sure you're not adding a secondary "user info has been loaded", as it'll make things inconsistent and more difficult to maintain.
`userInfo` when truthy is all we need to know to determine whether the user has logged in, we have activated them, and we've pulled back the latest info for them.

Reply:
i.e) Filter has selected the appropriate className to mark as active, and has the state synced up to the user preferences